Clarify use of null in lattice_vectors (#231)
Materials-Consortia/OPTiMaDe: optimade.rst

  - MUST be a list of three vectors *a*, *b*, and *c*, where each of the vectors MUST BE a list of the vector's coordinates along the x, y, and z Cartesian coordinates.
    (Therefore, the first index runs over the three lattice vectors and the second index runs over the x, y, z Cartesian coordinates).
  - For databases that do not define an absolute Cartesian system (e.g., only defining the length and angles between vectors), the first lattice vector SHOULD be set along *x* and the second on the *xy*-plane.

>    - This property MUST be an array of dimensions 3 times 3 regardless of the elements of :property:`dimension\_types`.

Reviewer: ```suggestion
  - This property MUST be an array of dimensions 3 times 3 regardless of the elements of :property:`dimension\_types`.
```

>      The vectors SHOULD by convention be chosen so the determinant of the :property:`lattice_vectors` matrix is different from zero.

Reviewer: ```suggestion
    The vectors SHOULD by convention be chosen so the determinant of the :property:`lattice_vectors` matrix is different from zero.
```

>      The vectors in the non-periodic directions have no significance beyond fulfilling these requirements.

Reviewer: ```suggestion
    The vectors in the non-periodic directions have no significance beyond fulfilling these requirements.
```

  - MUST be a list of three vectors *a*, *b*, and *c*, where each of the vectors MUST BE a list of the vector's coordinates along the x, y, and z Cartesian coordinates.
    (Therefore, the first index runs over the three lattice vectors and the second index runs over the x, y, z Cartesian coordinates).
  - For databases that do not define an absolute Cartesian system (e.g., only defining the length and angles between vectors), the first lattice vector SHOULD be set along *x* and the second on the *xy*-plane.

>    - This property MUST be an array of dimensions 3 times 3 regardless of the elements of :property:`dimension\_types`. The vectors SHOULD by convention be chosen so the determinant of the :property:`lattice_vectors` matrix is different from zero. The vectors in the non-periodic directions have no significance beyond fulfilling these requirements.

Reviewer: ```suggestion
   - This property MUST be an array of dimensions 3 times 3 regardless of the elements of :property:`dimension\_types`.
     The vectors SHOULD by convention be chosen so the determinant of the :property:`lattice_vectors` matrix is different from zero.
     The vectors in the non-periodic directions have no significance beyond fulfilling these requirements.
```

>   - All three elements of the inner lists of floats MAY be :val:`null` for non-periodic dimensions, i.e., those dimensions for which :property:`dimension\_types` is 0.

Reviewer: ```suggestion
  - All three elements of the inner lists of floats MAY be :val:`null` for non-periodic dimensions, i.e., those dimensions for which :property:`dimension\_types` is :val:`0`.
```

    (Therefore, the first index runs over the three lattice vectors and the second index runs over the x, y, z Cartesian coordinates).
  - For databases that do not define an absolute Cartesian system (e.g., only defining the length and angles between vectors), the first lattice vector SHOULD be set along *x* and the second on the *xy*-plane.

>   - This property MUST be an array of dimensions 3 times 3 regardless of the elements of property `dimension_types`_. The vectors SHOULD by convention be chosen so the determinant of the :property:`lattice_vectors` matrix is different from zero. The vectors in the non-periodic directions have no significance beyond fulfilling these requirements.

Reviewer 2: Just a minor suggestion for consistency in this block.

```suggestion
   - This property MUST be an array of dimensions 3 times 3 regardless of the elements of :property:`dimension\_types`. The vectors SHOULD by convention be chosen so the determinant of the :property:`lattice_vectors` matrix is different from zero. The vectors in the non-periodic directions have no significance beyond fulfilling these requirements.
```

>   - All three elements of the inner lists of floats MAY be :val:`null` for nonperiodic dimensions, i.e., those dimensions for which :property:`dimension\_types` is 0.

Reviewer 2: Elsewhere in the spec we've hyphenated "non-periodic":

```suggestion
  - All three elements of the inner lists of floats MAY be :val:`null` for non-periodic dimensions, i.e., those dimensions for which :property:`dimension\_types` is 0.
```
